microsoft-365/compliance/retention-cmdlets.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Identify the available PowerShell cmdlets for retention"
+f1.keywords:
+- NOCSH
+ms.author: cabailey
+author: cabailey
+manager: laurawi
+ms.date:
+audience: Admin
+ms.topic: reference
+ms.service: O365-seccomp
+ms.localizationpriority: normal
+ms.collection:
+- M365-security-compliance
+- SPO_Content
+- m365initiative-compliance
+description: Identify the PowerShell cmdlets for retention that support configuration at-scale, automation, or might be needed for advanced configuration scenarios.
+---
+
+# PowerShell cmdlets for retention policies and retention labels
+
+>*[Microsoft 365 licensing guidance for security & compliance](/office365/servicedescriptions/microsoft-365-service-descriptions/microsoft-365-tenantlevel-services-licensing-guidance/microsoft-365-security-compliance-licensing-guidance).*
+
+[!include[Purview banner](../includes/purview-rebrand-banner.md)]
+
+Use the following sections to identify the main PowerShell cmdlets that are available for retention policies and retention labels that you might need for configuration at-scale, automated scripts, or advanced configuration scenarios. For the full list of  cmdlets, see the [policy-and-compliance-retention list](/powershell/module/exchange#policy-and-compliance-retention) from the PowerShell documentation.
+
+Before you use these cmdlets, you must first [connect to Office 365 Security & Compliance Center PowerShell](/powershell/exchange/connect-to-scc-powershell).
+
+In the descriptions that follow, a policy for retention can refer to a retention policy (no labels), or a retention label policy. Each policy defines whether it's static or adaptive and the locations for the policy to be applied. The policy then requires one rule to complete the configuration.
+
+For example:
+- A retention policy needs a rule that defines the retention settings, such as retain for five years and then delete.
+
+When you use retention labels, these contain the retention settings and their policies need different rules:
+- A retention label policy that you publish needs a rule that defines which labels should be displayed in apps.
+- An auto-apply retention label policy needs a rule that defines the label to apply and the conditions for applying the label.
+
+## Retention cmdlets for most locations
+
+Use the cmdlets in the following table when the locations are **Exchange email**, **SharePoint sites**, **OneDrive accounts**, **Microsoft 365 Groups**, **Skype for Business**, **Exchange public folders**, **Teams chat messages**, or **Teams channel messages**.
+
+Don't use these cmdlets when the locations are for Teams private channel messages, Yammer user messages, or Yammer community messages. These locations have alternative cmdlets that are identified in the [next section](#retention-cmdlets-specific-to-teams-private-channels-and-yammer).
+
+|Cmdlet|Description|Applicable locations|
```

Comment:
Please look at the rich diff (preview) - the following two tables do not render correctly. Kindly correct the formatting.